06_System_Hacking.md

```diff
@@ -221,23 +221,23 @@ The function of salting is to defeat the Dictionary Attacks and Rainbow Table at
 
 ## Escalating Privileges
 
-The main goal is to get a high-level access to the system.
+The main goal is to get high-level access to the system.
 
 ### Horizontal Privileges Escalation
 
-The attacker attempts to gain access to user that has same set of privileges. 
+The attacker attempts to gain access to a user that has the same set of privileges. 
 
 ### Vertical Privileges Escalation
 
 The attacker attempts to escalate privileges to a higher level.
-Vertical privileges occurs when attacker is trying to gain access to the Administrator account.
-Higher privileges allow attacker to access sensitive information, modify files and execute programs.
+Vertical privileges occur when an attacker is trying to gain access to the Administrator account.
+Higher privileges allow an attacker to access sensitive information, modify files, and execute programs.
 
 ### Privilege Escalation using DLL Hijacking
 
 Applications need Dynamic Link Libraries (DLL) to run.
-In Windows, most of the application search for DLL in directories, instead of using the full qualified path.
-The Attacker replace the DLL to a malicious one.
+In Windows, most of the application search for DLL in directories, instead of using the full, qualified path.
```

Comment:
Both wrong (mine and yours). The correct would be "fully qualified path". See: https://en.wikipedia.org/wiki/Fully_qualified_name#Path_names